### ISSUE TITLE
Is Structural Optimization Possible with MACE? Looking for Sample Code

### ISSUE THREAD

**Issue by bishopfunc** [ACEsuit/mace#608]
**Is your feature request related to a problem? Please describe.**
Is it possible to perform structural optimization using MACE? Since MACE can be used for molecular dynamics, I feel like it should be theoretically possible. Are there any sample codes available for this?

**Comment by gabor1** [comment]
Yes of course. Anything that you can do with forces and energies is doable. We have plugins to LAMMPS and ASE, which both allow you to do geometry optimisation and MD. They have lots of example scripts and tutorials.

**Comment by gabor1** [comment]
https://wiki.fysik.dtu.dk/ase/ase/optimize.html

**Comment by gabor1** [comment]
And you can see how to instantiate a MACE calculator in ASE in our docs
